doc/list.rst

```diff
@@ -3113,6 +3113,8 @@ List of Available Rules
 
    *warning risky* Risky when using ``->bindTo`` on lambdas without referencing to ``$this``.
```

Comment:
I belive this rule is much more risky, as `parent::` or `ParentClass::` are `$this` instance calls and cannot be detected using single file/no static analysis.

Reply:
would you like to raise the PR to extend the description? 🙏🏻

Reply:
actually, the `parent::` case is already covered and we do not add `static` in such cases: https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/blob/master/tests/Fixer/FunctionNotation/StaticLambdaFixerTest.php#L321

not sure about `ParentClass::` explicitly, not sure if there is good way to detect (on tokenized level only) that `ParentClass::foo` is an actual parent or a static method

Reply:
So the docs would be nice to be updated to ...```on lambdas without referencing to ``$this``/``parent::``. ```